src/VoteProxy2.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.4.24;
+
+import 'ds-token/token.sol';
+import 'ds-chief/chief.sol';
+
+contract VoteProxy2 {
+    address public cold;
+    address public hot;
+    DSToken public GOV;
+    DSToken public IOU;
+    DSChief chief;
+
+    constructor(DSChief chief_, address cold_, address hot_) {
+        chief = chief_;
+        cold = cold_;
+        hot = hot_;
+        GOV = chief.GOV();
+        IOU = chief.IOU();
+        GOV.approve(chief, uint256(-1));
+        IOU.approve(chief, uint256(-1));
+    }
+
+    modifier onlyCold() { require(msg.sender == cold); _ }
+    modifier hotOrCold() { require(msg.sender == cold || msg.sender == hot); _ }
+
+    function vote(bytes32 slate)
+        hotOrCold
+    {
+        chief.vote(slate);
+    }
+    function lock(uint256 wad)
+        hotOrCold
+    {
+        chief.lock(wad);
+    }
+    function free(uint256 wad)
+        hotOrCold
+    {
+        chief.free(wad);
+    }
+    function release(uint256 wad)
+        onlyCold
```

Comment:
I think this should be hotOrCold.  If you have a paper wallet as `cold`, sending MKR back to cold storage should not require burning the paper wallet.

If an attacker manages to compromise `hot`, the worst they can do is give the MKR back to `cold`, no harm done.

Reply:
Yeah, we can do that. Funny enough, hot and cold wallets end up with identical access!

Reply:
Yeah, which means we can drop one of the modifiers.  😁